src/diffusers/models/unet_2d_blocks.py

```diff
@@ -830,14 +844,20 @@ def custom_forward(*inputs):
                     create_custom_forward(attn, return_dict=False),
                     hidden_states,
                     encoder_hidden_states,
+                    None,
+                    None,
```

Comment:
@pcuenca good question regarding `use_reentrant`. I don't know the answer, but worth pursuing in a separate ticket, in case it can avoid positional param mishaps in future.

I've accepted the commit suggestion (added param comments).